tests/lax_numpy_test.py

```diff
@@ -2299,25 +2299,26 @@ def testCompressMethod(self, shape, dtype, axis):
     self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker)
 
   @parameterized.named_parameters(jtu.cases_from_list(
-      {"testcase_name": "_axis={}_baseshape=[{}]_dtypes=[{}]".format(
-          axis, ",".join(str(d) for d in base_shape),
+      {"testcase_name": "_axis={}_dtype={}_baseshape=[{}]_argdtypes=[{}]".format(
+          axis, np.dtype(dtype).name, ",".join(str(d) for d in base_shape),
```

Comment:
`np.dtype(None).name` will result in `float64`, which is misleading. Maybe use something like `dtype and np.dtype(dtype).name` ?